checklist/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "amp-checklist-bot",
+  "author": "The AMP HTML Authors",
+  "version": "0.0.1",
+  "private": true,
+  "repository": "http://github.com/ampproject/amp-github-apps",
+  "description": "A GitHub app which posts checklists on pull requests.",
+  "engines": {
+    "node": "^12.13.1"
+  },
+  "scripts": {
+    "lint": "eslint **/*.ts",
+    "fix": "eslint **/*.ts --fix",
+    "lint-conflicts": "eslint --print-config app.ts | eslint-config-prettier-check",
+    "build": "tsc",
+    "build:watch": "tsc -w --p tsconfig.json",
+    "start": "probot run ./dist/app.js",
+    "dev": "nodemon",
```

Comment:
Does this work if you try to run it? :stuck_out_tongue:

Reply:
Yes :)